feat: Add dependency dashboard label configuration (#10601)
renovatebot/renovate: docs/usage/configuration-options.md

## dependencyDashboardHeader

## dependencyDashboardLabels

> Note: the labels will only be updated when the dependency dashboard issue would have been updated for its content and/or title. You are also not allowed to edit the labels: Renovate will restore them on each run.

Reviewer: ```suggestion
The labels only get updated when the Dependency Dashboard issue updates its content and/or title.
You are not allowed to edit the labels, as Renovate bot restores the labels on each run.
```

Not sure if you're not allowed/able to make changes, or if its just pointless to edit the labels, because Renovate throws away your changes?

Author: Pointless would be the better term indeed, I've adjusted it

Configure this option if you prefer a different title for the Dependency Dashboard.

## dependencyDashboardLabels

> 

Reviewer: I think you mean to add text here on how to use the feature? 😉

Author: If I'm not mistaken, that is automatically generated from the definitions?

https://github.com/renovatebot/renovate/pull/10601/files#diff-5a0608bb8d9d5dbdb218a0169373ad3514241002fba8ca5299507a2b32b87c9cR422-R428

Reviewer: I think you're right! 😄
